YoCode/GitCheck.cs

```diff
@@ -8,18 +8,14 @@ namespace YoCode
 {
     public class GitCheck
     {
-        public string GIT_INSTALL_DIRECTORY { get; set; } = @"C:\Program Files\Git";
-        //public string REPOSITORY_PATH = @"..\..\..\..\..\YoCode";
         public string REPOSITORY_PATH = @"C:\Users\ukmzil\source\repos\Tests Sent by People\Real\simon-jones";
 
-        string output { get; set; }
-        string lastAuthor { get; set; }
-        bool result { get; set; }
-
+        public string Output { get; set; }
+        public string LastAuthor { get; set; }     
+        public bool GitUsed => ExecuteTheCheck();
```

Comment:
This is going to result in `ExecuteTheCheck` running every time `GitUsed` is used in the code. It would be better to do something to make sure it only runs once I think but it's not super important at the moment.